pdns/lua-record.cc

```diff
@@ -952,7 +960,7 @@ std::vector<shared_ptr<DNSRecordContent>> luaSynth(const std::string& code, cons
         ret.push_back(DNSRecordContent::mastermake(qtype, QClass::IN, content ));
     }
   } catch(std::exception &e) {
-    g_log<<Logger::Error<<"Lua record reported: "<<e.what();
+    g_log << Logger::Info << "Lua record reported: " << e.what();
```

Comment:
I just wrote this into my local copy, it would conflict with this branch, so I'll leave it here:
```
    g_log<<Logger::Error<<"Lua record ("<<query<<"|"<<QType(qtype).getName()<<") reported: "<<e.what();
```

Can you take that change? I have no opinion on Info vs. Error.

Reply:
I've pushed this to your branch.